### PR TITLE
Fix request path attribute in HTTP docs

### LINKED ISSUE #20433: Logging attribute %R / %{RELATIVE_PATH}  does not work
### Describe the bug

The logging attribute as described in the Quarkus documentation, https://quarkus.io/guides/http-reference#configuring-http-access-logs , does not work.

### Expected behavior

I expect the parameter to work as documented

### Actual behavior

When starting the application, errors are logged and attributes are not logged properly:

```
2021-09-28 20:44:06,132 ERROR [io.qua.ver.htt.run.att.ExchangeAttributeParser] (Quarkus Main Thread) Unknown token %R
2021-09-28 20:44:06,137 ERROR [io.qua.ver.htt.run.att.ExchangeAttributeParser] (Quarkus Main Thread) Unknown token %{RELATIVE_PATH}
```

### How to Reproduce?

Create a Quarkus application, and set the following application properties:

```
quarkus.http.access-log.enabled=true
quarkus.http.access-log.pattern=%R %{RELATIVE_PATH}
```

Start the application, and the error will appear in the log

### Output of `uname -a` or `ver`

Darwin Kernel Version 20.4.0: Thu Apr 22 21:46:47 PDT 2021; root:xnu-7195.101.2~1/RELEASE_X86_64 x86_64

### Output of `java -version`

openjdk version "11.0.11" 2021-04-20

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

2.2.3

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

Apache Maven 3.8.2

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by geoand**
Fixes: #20433

### CODE REVIEW

**Comment by magnusvage** [comment]
@geoand %R won't work, so either it should be removed from the docs or added as an attribute

**Comment by geoand** [comment]
Lemme check

**Comment by geoand** [comment]
PR updated

File: extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/attribute/RequestPathAttribute.java [hunk 1]
```diff
@@ -5,6 +5,7 @@
 public class RequestPathAttribute implements ExchangeAttribute {
 
     public static final String REQUEST_PATH = "%{REQUEST_PATH}";
+    public static final String REQUEST_PATH_SHORT = "%{R}";
```
Comment (magnusvage) [inline extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/attribute/RequestPathAttribute.java:8]: @geoand should be `= "%R";` for consistency

Comment (geoand) [reply extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/attribute/RequestPathAttribute.java:8]: Yeah, you are right. Thanks for pointing it out

**Review by stuartwdouglas — APPROVED** [review]

**Pull request merged**